### PR TITLE
Quick cleanup

### PR DESCRIPTION
**Pull request by mathieu-benoit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,19 +38,17 @@ This reference architecture implementation uses Terraform. You will need to do t
 
 3. Ensure you are logged in with `gcloud` (See: [gcloud auth application-default login](https://cloud.google.com/sdk/gcloud/reference/auth/application-default)).
 
-   You will need to ensure your Google Cloud account has appropriate permissions on the project you wish to provision in (see [Required Google Cloud Permissions](#required-google-cloud-permissions)).
```
Comment (mathieu-benoit) [inline README.md:41]: No dedicated section, so removing it?

File: README.md [hunk 2]
```diff
@@ -38,19 +38,17 @@ This reference architecture implementation uses Terraform. You will need to do t
...
+   You will need to ensure your Google Cloud account has appropriate permissions on the project you wish to provision in.
 
-4. Ensure the [Kubernetes Engine API](https://console.cloud.google.com/marketplace/product/google/container.googleapis.com) is enabled for your project.
```
Comment (mathieu-benoit) [inline README.md:43]: Now in Terraform: `modules.base.main.tf`

File: modules/gke/gsas.tf [hunk 3]
```diff
@@ -28,13 +28,7 @@ resource "google_project_iam_member" "gke_admin" {
   member  = "serviceAccount:${google_service_account.gke_cluster_access.email}"
 }
 
-resource "google_project_iam_member" "gke_logging_viewer" {
```
Comment (mathieu-benoit) [inline modules/gke/gsas.tf:31]: Not needed because using `logging-k8s`

File: modules/gke/providers.tf [hunk 4]
```diff
@@ -9,9 +9,9 @@ terraform {
     helm = {
       source = "hashicorp/helm"
     }
-    http = {
```
Comment (mathieu-benoit) [inline modules/gke/providers.tf:12]: Not used by default, so let's comment to avoid `terraform init` to resolve/download it?

**Comment by mathieu-benoit** [comment]
Ready for your review, thanks!

**Review by johanneswuerbach — APPROVED** [review]

**Pull request merged**